whitepapers/Tables.md

```diff
@@ -1,5 +1,114 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| | | ✓ | ✓ | ✓ | | |
 
+#### Other indices
+* `{column_name}`, `{column_name}`, ... [(unique)]
+* ...
+
+## Data
+
+### SourceUniqueIdentifier (SUID)
+Identifier for a specific document from a specific source.
+
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| `source_doc_id` | text |  |  |  |  | Identifier given to the document by the source |
+| `ingest_config_id` | int |  |  | ✓ |  | IngestConfig used to ingest the document |
```

Comment:
Could `source_doc_id` just be `identifier`, having non-FK's ending `_id` feels a bit hairy?

I think it makes sense to link to source here. If we extract the same identifier they will be the same record(s).
And many times when the harvester changes the identifier changes as well.

Thoughts?